elements/rh-blockquote/rh-blockquote.ts

```diff
@@ -0,0 +1,74 @@
+import { LitElement, html, css, svg } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { pfelement } from '@patternfly/pfe-core/decorators.js';
+
+import styles from './rh-blockquote.css';
+
+/**
+ * A blockquote for displaying quote, author, and author title.
+ *
+ *
+ * @summary  A blockquote for displaying quote, author, and author title.
+ *
+ *
+ * @slot quote   - Provide a quote for the blockquote
+ * @slot author  - Provide an author for the blockquote
+ * @slot title   - Provide an author title for the blockquote
+ *
+ */
+@customElement('rh-blockquote') @pfelement()
+export class RhBlockquote extends LitElement {
+  static readonly styles = styles;
+
+  @property({ type: String }) title = 'Blockquote';
+
+  /**
+   * Set the theme of the blockquote. Possible values are:
+   * - `light` (default)
+   * - `dark`
+   */
+  @property({ reflect: true, attribute: 'theme' })
```

Comment:
```suggestion
  @property({ reflect: true, attribute: 'color-palette' })
```
Since the nature and timeline of our moving PFE context to RHDS, let's just implement this by hand, assuming that we'll keep `color-palette` (and `on`) as attribute names